easybuild/easyconfigs/l/libgpuarray/libgpuarray-0.7.6-fosscuda-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,49 @@
+easyblock = 'CMakeMake'
+
+name = 'libgpuarray'
+version = '0.7.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeplearning.net/software/libgpuarray/'
+description = "Library to manipulate tensors on the GPU."
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = ['https://github.com/Theano/libgpuarray/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['887b6433a30282cb002117da89b05812c770fd9469f93950ff3866ddd02bfc64']
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Mako', '1.1.0'),
+    ('NCCL', '2.4.8'),
+]
+
+configopts = '-DCMAKE_BUILD_TYPE=Release'
```

```suggestion
build_type = 'Release'
```

see https://github.com/easybuilders/easybuild-easyblocks/blob/85c2f77c2243d244fa21eaac40de960018e3aa3f/easybuild/easyblocks/generic/cmakemake.py#L76